CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - Fixed [#649](https://github.com/JabRef/jabref/issues/649): Key bindings are now working in the preview panel
 - Fixed [#410](https://github.com/JabRef/jabref/issues/410): Find unlinked files no longer freezes when extracting entry from PDF content
 - Fixed [#936](https://github.com/JabRef/jabref/issues/936): Preview panel is now updated when an entry is cut/deleted
+- Fixed Bug introduced with Commit [b3aee00](https://github.com/JabRef/jabref/commit/b3aee00bae8b589aa1252ef182f8b3293d06bd69): Entry is now correctly removed from BibDatabase  
```

Comment:
There is no need for a changelog entry in this case at all: The bug does not appear in a released version. Development versions may be broken. The text is OK for the commit. For a changelog, only the text is OK (without the commit id). However, the commit happend **after** the last release of JabRef and thus did not appear to the end user. If an enduser wondered, he would report an issue. Since no issue has been reported, one does not need to create a "fake" issue.

Nevertheless, it might be that #991 could be fixed with that PR.